schema-derivatives-perpfutures.graphql

```diff
@@ -1658,7 +1652,7 @@ type Liquidate implements Event @entity(immutable: true) {
   liquidator: Account!
 
   " Account that got liquidated "
-  liquidatee: Account!
```

Comment:
`Liquidate` is required to have an `account` field since it implements `Event` entity